homeassistant/components/reolink/__init__.py

```diff
@@ -90,7 +90,11 @@ async def async_device_config_update():
         update_interval=timedelta(seconds=DEVICE_UPDATE_INTERVAL),
     )
     # Fetch initial data so we have data when entities subscribe
-    await coordinator_device_config_update.async_config_entry_first_refresh()
+    try:
+        await coordinator_device_config_update.async_config_entry_first_refresh()
+    except ConfigEntryNotReady as err:
+        await host.stop()
+        raise err
```

Comment:
Just `raise` is enough to re-raise the exception.

Reply:
Thanks @MartinHjelmare
see https://github.com/home-assistant/core/pull/87224